diag/dtc/controller/dtc_controller.cpp

```diff
@@ -27,7 +27,7 @@ const uint8_t &dtc_status, const std::vector<uint8_t> &payload) {
     DtcHeader hdr{dtc_error_id, dtc_status};
     std::vector<uint8_t> data = this->factory_.GetBuffer(
             std::make_shared<DtcHeader>(hdr), payload);
-    return this->sock_.Transmit("0x0101", 0, data);
+    return this->sock_.Transmit("SIMBA.DIAG.DTC.0x0101", 0, data);
```

Comment:
tutaj nie jest potrzebny ten dopis bo i tak jest tylko jeden socket z dtc
```suggestion
    return this->sock_.Transmit("SIMBA.DIAG.DTC", 0, data);
```